Index: app/src/main/res/layout/activity_main.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/activity_main.xml b/app/src/main/res/layout/activity_authentication.xml
rename from app/src/main/res/layout/activity_main.xml
rename to app/src/main/res/layout/activity_authentication.xml
--- a/app/src/main/res/layout/activity_main.xml	(revision 3df9e49e3d204b6639f6c58ad770f8a285bc1425)
+++ b/app/src/main/res/layout/activity_authentication.xml	(date 1676132629610)
@@ -4,15 +4,16 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".presentation.AuthenticationActivity">
 
     <TextView
+        android:id="@+id/textView"
+        style="@style/TextAppearance.MaterialComponents.Headline3"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginTop="56dp"
+        android:text="@string/app_title"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
-
 </androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
Index: app/src/main/java/edu/tamykyy/lychat/MainActivity.java
===================================================================
diff --git a/app/src/main/java/edu/tamykyy/lychat/MainActivity.java b/app/src/main/java/edu/tamykyy/lychat/MainActivity.java
deleted file mode 100644
--- a/app/src/main/java/edu/tamykyy/lychat/MainActivity.java	(revision 3df9e49e3d204b6639f6c58ad770f8a285bc1425)
+++ /dev/null	(revision 3df9e49e3d204b6639f6c58ad770f8a285bc1425)
@@ -1,14 +0,0 @@
-package edu.tamykyy.lychat;
-
-import androidx.appcompat.app.AppCompatActivity;
-
-import android.os.Bundle;
-
-public class MainActivity extends AppCompatActivity {
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
-    }
-}
\ No newline at end of file
Index: app/src/main/res/values/strings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><resources>\n    <string name=\"app_name\">LyChat</string>\n</resources>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/values/strings.xml b/app/src/main/res/values/strings.xml
--- a/app/src/main/res/values/strings.xml	(revision 3df9e49e3d204b6639f6c58ad770f8a285bc1425)
+++ b/app/src/main/res/values/strings.xml	(date 1676132629645)
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">LyChat</string>
+    <string name="app_title">lyChat</string>
 </resources>
\ No newline at end of file
Index: .idea/render.experimental.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/render.experimental.xml b/.idea/render.experimental.xml
new file mode 100644
--- /dev/null	(date 1676132546475)
+++ b/.idea/render.experimental.xml	(date 1676132546475)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RenderSettings">
+    <option name="useLiveRendering" value="false" />
+  </component>
+</project>
\ No newline at end of file
Index: app/src/main/java/edu/tamykyy/lychat/presentation/AuthenticationActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/tamykyy/lychat/presentation/AuthenticationActivity.java b/app/src/main/java/edu/tamykyy/lychat/presentation/AuthenticationActivity.java
new file mode 100644
--- /dev/null	(date 1676132519995)
+++ b/app/src/main/java/edu/tamykyy/lychat/presentation/AuthenticationActivity.java	(date 1676132519995)
@@ -0,0 +1,20 @@
+package edu.tamykyy.lychat.presentation;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import edu.tamykyy.lychat.R;
+
+public class AuthenticationActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_authentication);
+
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().hide();
+        }
+    }
+}
\ No newline at end of file
Index: app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"utf-8\"?>\n<manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\n    xmlns:tools=\"http://schemas.android.com/tools\">\n\n    <application\n        android:name=\".app.App\"\n        android:allowBackup=\"true\"\n        android:dataExtractionRules=\"@xml/data_extraction_rules\"\n        android:fullBackupContent=\"@xml/backup_rules\"\n        android:icon=\"@mipmap/ic_launcher\"\n        android:label=\"@string/app_name\"\n        android:supportsRtl=\"true\"\n        android:theme=\"@style/Theme.LyChat\"\n        tools:targetApi=\"31\">\n        <activity\n            android:name=\".MainActivity\"\n            android:exported=\"true\">\n            <intent-filter>\n                <action android:name=\"android.intent.action.MAIN\" />\n\n                <category android:name=\"android.intent.category.LAUNCHER\" />\n            </intent-filter>\n        </activity>\n    </application>\n\n</manifest>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/AndroidManifest.xml b/app/src/main/AndroidManifest.xml
--- a/app/src/main/AndroidManifest.xml	(revision 3df9e49e3d204b6639f6c58ad770f8a285bc1425)
+++ b/app/src/main/AndroidManifest.xml	(date 1676131346618)
@@ -13,7 +13,7 @@
         android:theme="@style/Theme.LyChat"
         tools:targetApi="31">
         <activity
-            android:name=".MainActivity"
+            android:name=".presentation.AuthenticationActivity"
             android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
